src/views/Accessibility/TestingTemplates/index.tsx

```diff
@@ -0,0 +1,198 @@
+            )}`}
+          >
+            {t('testingTemplates.remediationPlanSection.heading')}
+          </Link>
+        </li>
+      </ul>
+    </div>
+  );
+  const testPlanSection = (
+    <div>
+      <h2
+        id={`testingTemplates_${t('testingTemplates.testPlanSection.heading')}`}
+      >
+        {t('testingTemplates.testPlanSection.heading')}
+      </h2>
+      <p>{t('testingTemplates.testPlanSection.description')}</p>
+      <ul className="accessibility-testing-templates__test-plan-list">
+        {testPlanList.map(item => (
+          <li key={item}>{item}</li>
+        ))}
+      </ul>
+    </div>
+  );
+  const remediationPlanSection = (
+    <div>
+      <h2
+        id={`testingTemplates_${t(
+          'testingTemplates.remediationPlanSection.heading'
+        )}`}
+      >
+        {t('testingTemplates.remediationPlanSection.heading')}
+      </h2>
+      <p>{t('testingTemplates.remediationPlanSection.description')}</p>
+      <ul className="accessibility-testing-templates__remediation-plan-list">
+        {remediationPlanList.map(item => (
+          <li key={item}>{item}</li>
+        ))}
+      </ul>
+    </div>
+  );
+
+  const downloadVPAT = (
+    <div className="accessibility-testing-templates__downloadBox">
+      <h3 className="margin-top-0">
+        {t('testingTemplates.vpatSection.subSection.downloadVPAT.heading')}
+      </h3>
+      <p>
+        <Link
+          href="https://www.itic.org/policy/accessibility/vpat"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          {t(
+            'testingTemplates.vpatSection.subSection.downloadVPAT.line1.linkText'
+          )}
+        </Link>
+        {` `}
+        {t(
+          'testingTemplates.vpatSection.subSection.downloadVPAT.line1.otherText'
+        )}
+      </p>
+      <p>
+        <i className="fa fa-exclamation-circle fa-2x" />
+        {` `}
+        {t('testingTemplates.vpatSection.subSection.downloadVPAT.line2.text')}
+      </p>
+      <p>
+        <Link
+          href="https://www.youtube.com/watch?v=kAkSV9xiJ1A"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          {t(
+            'testingTemplates.vpatSection.subSection.downloadVPAT.line3.linkText'
+          )}
+        </Link>
+        {` `}
+        {t(
+          'testingTemplates.vpatSection.subSection.downloadVPAT.line3.otherText'
+        )}
+      </p>
+    </div>
+  );
+
+  const vpatSection = (
+    <div>
+      <h2 id={`testingTemplates_${t('testingTemplates.vpatSection.heading')}`}>
+        {t('testingTemplates.vpatSection.heading')}
+      </h2>
+      <p>{t('testingTemplates.vpatSection.description')}</p>
+      <h3>{t('testingTemplates.vpatSection.subSection.heading')}</h3>
+      <ul className="accessibility-testing-templates__vpat-list">
+        <li>{t('testingTemplates.vpatSection.subSection.item1.text')}</li>
+        {vpatConformanceLevels.map(level => (
+          <p key={level.name}>
```

Comment:
This might be a good place to leverage `dl`, `dt`, and `dd`.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl

tl;dr; These HTML tags are used to assign definitions to terms, which is what's happening here.